src/scss/settings/_variables.scss

```diff
@@ -41,8 +41,8 @@ $type: (
         narrow: (16, 20)
     ),
     large--scaleUp: (
-        default: (32, 40),
-        narrow: (18, 24)
+        default: (20, 24),
```

Comment:
The values no longer fit the name of this one.  Large is `24/28` so the name should change to `large--scaleDown` if it's there to transform that size down a little

Reply:
Also - any idea what currently uses this variable?

Reply:
It's used for the homepage strapline in the new hero slingshot work. Currently only in UK so I wanted to update it so GlobalWeb is inline. I'll update the name shortly.